lib/resources/http.rb

```diff
@@ -0,0 +1,51 @@
+# encoding: utf-8
+# copyright: 2017, Criteo
+# author: Guilhem Lettron
+# license: All rights reserved
+
+require 'requests'
+
+module Inspec::Resources
+  class Http < Inspec.resource(1)
+    name 'http'
+    desc 'Use the http InSpec audit resource to test http call.'
+    example "
+      describe http(http://localhost:8080/ping', auth: ['user, 'test'], params: {format: 'html'}) do
+        its('status') { should eq(200) }
+        its('body') { should eq('pong')}
+        its('Content-Type') { should include('text/html') }
+      end
+    "
+
+    # rubocop:disable ParameterLists
+    def initialize(url, method: 'GET', params: nil, auth: nil, headers: {}, data: nil)
+      @method = method
+      @url = url || 'http://localhost/'
+      @params = params
+      @auth = auth
+      @headers = headers
+      @data = data
+      @request = Requests.request(@method, @url,
```

Comment:
I am open for another gem. Unfortunately we cannot go with the current one. Since, once merged, this PR it is part of a maintained project, we need to be sure we get all PRs in a way that we can maintain them going forward. I am open for alternative solutions.